beancount_import/source/schwab_csv.py

```diff
@@ -176,8 +176,15 @@ class BrokerageAction(enum.Enum):
     MISC_CASH_ENTRY = "Misc Cash Entry"
     JOURNALED_SHARES = "Journaled Shares"
     SECURITY_TRANSFER = "Security Transfer"
+    EXPIRED = "Expired"
+    STOCK_MERGER = "Stock Merger"
+    BOND_INTEREST = "Bond Interest"
+    SHORT_TERM_CAP_GAIN = "Short Term Cap Gain"
+    LONG_TERM_CAP_GAIN = "Long Term Cap Gain"
 
 
+MergerSpecification = namedtuple('MergerSpecification', ['symbol', 'quantity', 'description'])
```

Comment:
Changed it to a dataclass. One odd thing is that quantity must be Optional[Decimal] for mypy to pass (because that's what _convert_decimal returns), but in reality it really cannot be None. This means in such situations we end up inserting asserts right before the creation of the dataclass. I don't know of a better pattern. Let me know if you do.